### PR TITLE
 LPS-140252 Update status fields always

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageLocalServiceImpl.java [hunk 1]
```diff
@@ -3334,6 +3334,10 @@ private WikiPage _updatePage(
 		page.setVersion(newVersion);
 		page.setMinorEdit(minorEdit);
 		page.setContent(content);
+		page.setSummary(summary);
+		page.setFormat(format);
+
+		page.setStatusByUserId(user.getUserId());
```
Comment (adolfopa) [inline modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageLocalServiceImpl.java:3340]: Doesn't this line need to be sorted?  Not sure if the service.xml imposes a weird ordering. If this is fine here, you need to drop the blank line before the call.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageLocalServiceImpl.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageLocalServiceImpl.java:3340]: You are right

The correct order is

```
<column name="status" type="int" />
<column name="statusByUserId" type="long" />
<column name="statusByUserName" type="String" uad-anonymize-field-name="fullName" />
<column name="statusDate" type="Date" />
```

Thanks

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Pull request closed without merging**